install_script_tools.sh

```diff
@@ -49,6 +49,7 @@ fi
 
 cd $PIHOME/$DEXTER/$LIB/$DEXTER/$SCRIPT
 sudo apt-get install build-essential libi2c-dev i2c-tools python-dev libffi-dev -y
-python autodetect_setup.py install
+python setup.py install
+python3 setup.py install
```

Comment:
For both commands `python` & `python3` I'd use the following format that first checks if the executable exists and then it executes it:
```
command -v python >/dev/null && python setup.py install
command -v python3 >/dev/null && python setup.py install
```
This should be a better solution that what we already have.

Reply:
Over ruled for now

Reply:
This over-rule did come after discussion between Robert and I